test/rulesets/Base/fastmath_able.jl

```diff
@@ -142,8 +140,8 @@ const FASTABLE_AST = quote
             Δz = randn(typeof(f(x, y)))
 
             # some tests struggle in fast_math mode to get accurasy so we relax it some.
-            frule_test(f, (x, Δx), (y, Δy); rtol=(is_fastmath_mode ? 1e-5 : 1e-7))
-            rrule_test(f, Δz, (x, x̄), (y, ȳ); rtol=(is_fastmath_mode ? 1e-5 : 1e-7))
+            frule_test(f, (x, Δx), (y, Δy))
```

Comment:
can we delete comment above too? (github doesn't seem to let me directly suggest that change)